astropy/nddata/mixins/ndarithmetic.py

```diff
@@ -479,7 +479,7 @@ def _arithmetic_wcs(self, operation, operand, compare_wcs, **kwds):
             The ``wcs`` of the first operand is returned.
         """
 
-        # ok, not really arithmetics but we need to check which wcs makes sense
+        # ok, not really arithmetic but we need to check which wcs makes sense
```

Comment:
The change `arithmetics` → `arithmetic` seems like a British vs. American English thing, so I put it all in a single commit that can be reverted if we want to.

Reply:
My spelling is a horrible mix of British and US English, having lived in US, CA, and UK, so don't ask me for these spell checks 😆 .

Reply:
Apparently it's not even a British thing; they do mathematics, but may only specialise in arithmetic. 😃